samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/CallingBot.cs

```diff
@@ -98,21 +108,69 @@ private async Task NotificationProcessor_OnNotificationReceivedAsync(Notificatio
             // https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/articles/index.html#answer-incoming-call-with-service-hosted-media
 
             graphLogger.CorrelationId = args.ScenarioId;
+            var callId = GetCallIdFromNotification(args);
+
             if (args.ResourceData is Call call)
             {
+
                 if (args.ChangeType == ChangeType.Created && call.State == CallState.Incoming)
                 {
-                    await callService.Answer(GetCallIdFromNotification(args), audioRecordingConstants.Speech, audioRecordingConstants.PleaseRecordYourMessage);
+                    await callService.Answer(callId, audioRecordingConstants.Speech, audioRecordingConstants.PleaseRecordYourMessage);
```

Comment:
Can you add comments to the if blocks, describing the case that is handled and what the bot does? e.g., new incoming call